src/main/resources/io/jenkins/plugins/audit/config/AuditLogConfiguration/config.jelly

```diff
@@ -0,0 +1,10 @@
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout"
```

Comment:
You need to add `<?jelly escape-by-default='true'?>` to the top. Security feature to avoid XSS and similar exploits.